backend/vaa-strapi/types/generated/components.d.ts

```diff
@@ -73,7 +73,8 @@ export interface LabelsViewTexts extends Schema.Component {
       Attribute.DefaultTo<'Tell Your Opinions'>;
     yourOpinionsDescription: Attribute.Text &
       Attribute.Required &
-      Attribute.DefaultTo<'Next, the app will ask your opinions on {{0}} statements about political issues and values, which the candidates have also answered. After you\'ve answered them, the app will find the candidates that best agree with your opinions. The statements are grouped into {{1}} categories. You can answer all of them or only select those you find important.'>;
+      // eslint-disable-next-line quotes
```

Comment:
Prettier wants the double quotes but ESLint wants them gone. I also can't commit this without the eslint-disable flag because of the pre-commit hook that automatically formats it incorrectly. I don't think fixing this is in the scope of this PR